pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -795,34 +795,33 @@ func setTrackerLabel(meta metav1.ObjectMeta, instance *v2vv1.VirtualMachineImpor
 func (r *ReconcileVirtualMachineImport) startVM(provider provider.Provider, instance *v2vv1.VirtualMachineImport, vmName types.NamespacedName) error {
 	vmi := &kubevirtv1.VirtualMachineInstance{}
 	err := r.client.Get(context.TODO(), vmName, vmi)
-	if err != nil {
-		if k8serrors.IsNotFound(err) {
-			if err = r.updateProgress(instance, progressStartVM); err != nil {
+	if err != nil && k8serrors.IsNotFound(err) {
```

Comment:
just for readability - I'd split this differently:
```
if err != nil {
  if k8serrors.IsNotFound(err) {
  ...
  } else {
    return err
  }
} else {
  // line 808-821
}
return nil
```

Reply:
Sure, we can do that.